doc/helpers/Input-extensions.md

```diff
@@ -13,7 +13,7 @@ Provides various attached properties for _input controls_, such as `TextBox` and
 | `AutoDismiss`          | `bool`       | Whether the soft keyboard will be dismissed when the enter key is pressed.                                                                        |
 | `AutoFocusNext`        | `bool`       | Whether the focus will move to the next focusable element when the enter key is pressed.\*                                                        |
 | `AutoFocusNextElement` | `Control`    | Sets the next control to focus when the enter key is pressed.\*                                                                                   |
-| `ReturnType`           | `ReturnType` | The type of return button on a soft keyboard for Android/iOS. It can be one of the following options: __Default, Done, Go, Next, Search, Send__.  |
+| `ReturnType`           | `InputReturnType` | The type of return button on a soft keyboard for Android/iOS. It can be one of the following options: __Default, Done, Go, Next, Search, Send__.  |
```

Comment:
We will need to add a section about these breaking changes to the upgrading docs. Currently we only have upgrading docs specific to **Material** Toolkit. As this change is in the core Uno.Toolkit.WinUI library, we should have a new general Upgrading Toolkit docs page and write there about the two breaking changes:

1. The switch from ReturnType to InputReturnType
2. The bumped dependency to Uno 6.0